lib/src/combined_wrappers/combined_iterable.dart

```diff
@@ -17,7 +17,7 @@ class CombinedIterableView<T> extends IterableBase<T> {
   /// The iterables that this combines.
   final Iterable<Iterable<T>> _iterables;
 
-  /// Creates a combined view of [iterables].
+  /// Creates a combined view of [_iterables].
```

Comment:
type is not public anyway, but the lint flags this reference, for obvious reasons